tests/e2e/support/step_definitions/api.js

```diff
@@ -291,6 +351,16 @@ function setRoleIntercepts() {
     },
   });
 
+  cy.intercept('GET', '/api/roles?name=not_SUPER_ADMINISTRATOR', {
```

Comment:
don't forget to add intercept for
`/api/roles?id=not_X&name=not_SUPER_ADMINISTRATOR`

Reply:
done